src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -80,11 +85,11 @@
 
     public static final class PanelRotator
     {
-        public static final int kBeamSensorUpID = -1;
-        public static final int kBeamSensorDownID = -1;
+        public static final int kBeamSensorUpID = 5;
+        public static final int kBeamSensorDownID = 6;
 
-        public static final int kExtendMotorID = -1;
-        public static final int kSpinMotorID = -1;
+        public static final int kExtendMotorID = 7;
```

Comment:
The PR still isn't working properly and I don't know why.

Look [here](https://github.com/Spartronics4915/2020-InfiniteRecharge/blob/master/src/main/java/com/spartronics4915/frc2020/Constants.java#L81) for what Constants _really_ should be.